src/Employer/Employer.Web/Orchestrators/VacanciesSearchSuggestionsOrchestrator.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections;
+
+namespace Esfa.Recruit.Employer.Web.Orchestrators
+{
+    public class VacanciesSearchSuggestionsOrchestrator
+    {
+        public const int MaxRowsInResult = 50;
+
+        private readonly IEmployerVacancyClient _employerVacancyClient;
+
+        public VacanciesSearchSuggestionsOrchestrator(IEmployerVacancyClient employerVacancyClient)
+        {
+            _employerVacancyClient = employerVacancyClient;
+        }
+
+        public async Task<IEnumerable<string>> GetSearchSuggestionsAsync(string searchTerm, string employerAccountId)
+        {
+            if (searchTerm == null || searchTerm.Trim().Length < 3) return Enumerable.Empty<string>();
+
+            var vacancies = await GetVacanciesAsync(employerAccountId);
+
+            var data = GetVacanciesPartialMatchingTitleSuggestions(searchTerm, vacancies)
```

Comment:
I think from this point downwards is the same as Provider.Web.  I wonder if we should create a shared service that does the filtering on a `IEnumerable<VacancySummary>`

Reply:
Did try it but it was becoming very complicated. Hence leaving it as is after our discussion.

Reply:
👍